Frends.AzureBlobStorage.UploadBlob/README.md

```diff
@@ -17,37 +17,6 @@ FRENDS Task for uploading a file to Azure.
 
 You can install the task via FRENDS UI Task View or you can find the nuget package from the following nuget feed 'Insert nuget feed here'.
 
-# Task
```

Comment:
https://github.com/FrendsPlatform/Frends.AzureBlobStorage/blob/ffc0f12d5b6c894ec772c84b73d701b00c13f815/Frends.AzureBlobStorage.UploadBlob/README.md?plain=1#L9-L14
These lines can be removed.